dbms/src/Storages/Transaction/PDTiKVClient.h

```diff
@@ -49,7 +51,7 @@ struct PDClientHelper
         {
             // In case we cost too much to update safe point from PD.
             std::chrono::time_point<std::chrono::system_clock> now = std::chrono::system_clock::now();
-            const auto duration = std::chrono::duration_cast<std::chrono::seconds>(now - safe_point_last_update_time);
+            const auto duration = std::chrono::duration_cast<std::chrono::seconds>(now - safe_point_last_update_time.load());
```

Comment:
Maybe using `*` is enough:

```suggestion
            const auto duration = std::chrono::duration_cast<std::chrono::seconds>(now - *safe_point_last_update_time);
```

Reply:
@breezewish there is no `operator *` for atomic type in C++ 🤔 https://en.cppreference.com/w/cpp/atomic/atomic